label_studio_ml/api.py

```diff
@@ -55,6 +55,7 @@ def _setup():
 @_server.route('/train', methods=['POST'])
 @exception_handler
 def _train():
+    logger.warning("Using deprecated method - switch to /webhook!")
```

Comment:
It's very unclear warning, we need something like:

"=> Warning: API /train is deprecated since Label Studio 1.4.1. ML backend used API /train for training previously, but since 1.4.1 LS backend and ML backend use webhooks for the training run. " 

Please correct if something is wrong.

Reply:
Fixed